front-end/src/shared/SideBarNav.js

```diff
@@ -0,0 +1,80 @@
+import React from 'react';
+import styled from 'styled-components';
+import Logo from '../svg/Logo';
+
+import {
+    BrowserRouter as Router,
+    Switch,
+    Route,
+    Link
+} from "react-router-dom";
+
+
+
+
+class SideBarNav extends React.Component {
+    render() {
+        return (
+            <LeftPanel>
```

Comment:
Can you please make sure that WebStorm is indenting according to the JavaScript convention with 2 spaces please? This looks to be done with 4.